integration/query_projection_test.go

```diff
@@ -63,61 +63,6 @@ func TestQueryProjection(t *testing.T) {
 	}
 }
 
-func TestQueryProjectionElemMatch(t *testing.T) {
-	setup.SkipForTigris(t)
-
-	t.Parallel()
-	providers := []shareddata.Provider{shareddata.Composites}
-	ctx, collection := setup.Setup(t, providers...)
-
-	// Comparison fails if we add below array to shareddata.
-	// TODO: move to compat https://github.com/FerretDB/FerretDB/issues/1569
-
-	_, err := collection.InsertMany(ctx, []any{
-		bson.D{
-			{"_id", "document-composite-2"},
-			{"v", bson.A{
-				bson.D{{"field", int32(42)}},
-				bson.D{{"field", int32(44)}},
-			}},
-		},
-	})
-	require.NoError(t, err)
-
-	for name, tc := range map[string]struct {
-		projection  any
-		expectedIDs []any
-	}{
-		"ElemMatch": {
-			projection: bson.D{{
-				"v",
-				bson.D{{"$elemMatch", bson.D{{"field", bson.D{{"$eq", 42}}}}}},
```

Comment:
Already moved in https://github.com/FerretDB/FerretDB/blob/main/integration/query_projection_compat_test.go#L60 and now shareddata has array numbers in composit `"array-numbers-asc": bson.A{int32(42), int32(43), int32(45)}`